include/circt/Dialect/SV/SVPasses.td

```diff
@@ -26,4 +26,16 @@ def RTLCleanup : Pass<"rtl-cleanup", "rtl::RTLModuleOp"> {
   let constructor = "circt::sv::createRTLCleanupPass()";
 }
 
+def RTLGreyBoxer : Pass<"rtl-grey-boxer", "mlir::ModuleOp"> {
```

Comment:
I'd recommend a more explicit name for this like "-stub-module-prototypes".  It isn't obvious what "grey boxing".